### DISCUSSION TITLE
I have iota from years ago. I have my seed phrase, but cannot sweep or migrate to new firefly. What do i do to get my crypto?

### DISCUSSION THREAD

**Discussion by Ren1895** [iotaledger/firefly#7672] [Feedback]
I have iota from years ago. I have my seed phrase, but cannot sweep or migrate to new firefly. What do i do to get my crypto?

**Comment by begonaalvarezd** [comment]
@Ren1895 do you have a 81 trinity seed? or do you have a 24 words mnemonic?

**Reply by cooolx** [reply]
I also meet this issue now, I have the 81 character but don't have the 24 words mnemonic, with firefly-desktop-1.7.2.AppImage, this support restore wallet with the seed, but will show a network connection error.
And version firefly-iota-desktop-2.0.3.AppImage not support restore wallet with the seed any more, how can I find my funds?
Thanks!

**Reply by begonaalvarezd** [reply]
The network got recently upgraded so version 1.7.2 doesnt work anymore as it is not compatible with the new network protocol. 

About the 81 chars seed migration, The current ability to auto-migrate tokens from the pre-Chrysalis IOTA network via the Firefly wallet have been stopped with the IOTA Stardust upgrade. Once IOTA smart contracts are available on the IOTA network, this feature will be restored and migrations can be processed automatically again.
We will provide a migration tool to resume legacy migrations, but its not finished yet
Firefly is not intended to provide migration anymore, but it will be a different tool. The Foundation will notify it when the tool is ready 🌷

**Reply by KaherdinTristan** [reply]
@begonaalvarezd 
_This thread was one of my first search results when googling "iota seed restore". So this thread already is, or might become quite relevant to redirect these kinds of questions._ 

- Where should end users looking for ways to recover "old" IOTA seed wallets/funds go? (url)

Best regards

**Reply by TheRabbitX** [reply]
@begonaalvarezd I am in the same situation and I really need the migration tool. I'll keep an eye on this thread and hope that the tool will be available soon. Thanks!

**Comment by javaherian1** [comment]
i have old   .kdbx file , but i cannot drag/drop into new iota wallet

**Comment by begonaalvarezd** [comment]
🔈 **We just released the new legacy migration tool** 🔈

Read about it here: https://blog.iota.org/iota-legacy-migration-tool/ 

If you face any issues please contact us over discord https://discord.iota.org/ or open an issue directly in the legacy tool repo https://github.com/iotaledger/legacy-migration-tool

Happy migration 🌷
